docs/features/network-security-controls/admin-network-policy.md

```diff
@@ -1,303 +1,1118 @@
+  network policy layer. If you don't care about interoperability with existing
+  network policies, then you probably don't need to worry about this feature.
+* Since `0-100` is the supported priority range, it is always better when designing
+  your admin policies to use a middle range like `30-70` by leaving some placeholder
+  priorities before and after. Even in the middle range, be careful to leave gaps so
+  that as your infrastructure requirements evolve over time, you are able to insert
+  new ANPs when needed at the right priority level. If you pack your ANPs, then you
+  might need to recreate all of them to accommodate any changes in the future.
+* Each rule can have upto 100 peers. All those peers are added to the same address-set.
+  It is for the best if selectors across multiple rules don't overlap so that different
+  address-sets don't have the same IPs in them. OVN has perf/scale limitations where
+  it doesn't do incremental processing at ovn-controller level in such cases. So any pod
+  level churn causes full processing of OVS flows. See [here](https://issues.redhat.com/browse/OCPBUGS-31385)
+  for more details.
+* `networks` peer in egress rule is meant to select external destinations. It is recommended
+  to use `namespaces` and `pods` peers to select in-cluster peers. Do not use `networks` for
+  podIP level matching.
+* Currently in order to express multi-tenant isolation, you need to do 1 ANP per tenant.
+  There is some future planned work upstream to make this easier for users. See multi tenant
+  isolation section for more details.
+
+## Known Limitations and Design Choices
+
+* Even if API supports upto 1000 as a value for `spec.priority`, **in OVN-Kubernetes
+  we only support upto 99 as a max value**. This is because of known limitation
+  in OVN level for `nbdb.ACL` priority field. See [this](https://bugzilla.redhat.com/show_bug.cgi?id=2175752)
+  for more details. However the number of admin policies in a cluster are usually expected
+  to be of a maximum of say 30-50 and not more than that based on use cases for
+  which this API was defined for. **Thus supported priority values in
+  OVN-Kubernetes are from 0 to 99**. When user tries to create an ANP with priority
+  greater than 99 an event `ANPWithUnsupportedPriority` will be emitted. The user has
+  to manually update the priority field so that it get's created correctly.
+* The `subject` of an admin network policy does NOT include support for
+  `host-networked` pods.
+* The `namespaces` and `pods` peers in admin network policy does NOT include
+  support for `host-networked` pods.
+* `nodes` peer can be specified only from `egress` rule. There are no ingress use
+  cases yet which is why this is not supported from `ingress` rule.
+* `networks` peer can be specified only from `egress` rule. There are no ingress use
+  cases yet which is why this is not supported from `ingress` rule.
+* Specifying `namedPorts` with `networks` peer is not supported.
```

Comment:
I think you should distinguish between "limitations/design choices of ovn-kubernetes's ANP implementation" and "limitations/design choices of the upstream API". (And maybe not even bother to list the latter.)

Reply:
I separated it, its good to explicitly list some things out even for the API even if its already part of the CRD description fields..